recipes/dash-core-components/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "dash-core-components" %}
+{% set filename = "dash_core_components" %}
+{% set version = "0.22.1" %}
+{% set sha256 = "2dbbc0121eab102d92f7b407099cb28818676f3753ea635fb29a9f8b9df0a118" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - dash_core_components
+    - dash_core_components.version
+
+about:
+  home: https://plot.ly/dash
+  dev_url: https://github.com/plotly/dash-core-components
+  license: MIT
+  license_family: MIT
```

Comment:
Can you manually package the license_file and add the following link as a comment so that if/when it gets merged, there is a visual reminder to update the feedstock later:
https://github.com/plotly/dash-core-components/pull/196

https://conda-forge.org/docs/meta.html#packaging-the-license-manually